public/scripts/tags.js

```diff
@@ -1417,8 +1439,21 @@ function sortTags(tags) {
  * @returns {number} The compare result
  */
 function compareTagsForSort(a, b) {
+    // default sort: alphabetical, case insensitive
     const defaultSort = a.name.toLowerCase().localeCompare(b.name.toLowerCase());
-    if (power_user.auto_sort_tags) {
+
+    // sort on number of entries
+    if (power_user.tag_sort_mode === 'by_entries') {
+
+        const countA = a.count !== undefined ? a.count : -1;
+        const countB = b.count !== undefined ? b.count : -1;
+
+        if (countA !== countB) {
+            return countB - countA;
+        }
+        return defaultSort;
```

Comment:
I think this whole block could be shortened. Though it doesn't matter that much. Just reads cleaner.


```suggestion
        return ((b.count || 0) - (a.count || 0)) ||  defaultSort;
```